flocker/control/httpapi.py

```diff
@@ -523,7 +551,8 @@ def get_containers_configuration(self):
         :return: A ``list`` of ``dict`` representing each of the containers
             that are configured to exist anywhere on the cluster.
         """
-        return list(containers_from_deployment(self.persistence_service.get()))
+        return list(
+            _containers_from_deployment(self.persistence_service.get()))
```

Comment:
I changed `containers_from_deployment` to be a private function to this file, made the width of this line 80 characters :(.

Made private since it's API is so terrible. (can't mutate result)